recipe/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "mixbox" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 13c618a36967a6906d09e9e5be952656c78279b0e9cb5527e9360416e4d1c057
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - lxml
+    - ordered-set
+    - python-dateutil
+    - weakrefmethod >=1.0.3
```

```suggestion
    - weakrefmethod >=1.0.3  # [py<34]
```
This is technically not required. They support [weakref](https://docs.python.org/3/library/weakref.html), see https://inspector.pypi.io/project/mixbox/1.0.5/packages/ff/b1/38a53aa8a059f09a06881e948cccff067c664e88f8a14cbfef8c5c29fac9/mixbox-1.0.5.tar.gz/mixbox-1.0.5/mixbox/signals.py#line.19.

---

ok